packages/wrangler/src/publish.ts

```diff
@@ -468,21 +468,30 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 		if (!props.dryRun) {
 			// Upload the script so it has time to propagate.
 			// We can also now tell whether available_on_subdomain is set
-			available_on_subdomain = (
-				await fetchResult<{ available_on_subdomain: boolean }>(
-					workerUrl,
-					{
-						method: "PUT",
-						body: createWorkerUploadForm(worker),
-					},
-					new URLSearchParams({
-						include_subdomain_availability: "true",
-						// pass excludeScript so the whole body of the
-						// script doesn't get included in the response
-						excludeScript: "true",
-					})
-				)
-			).available_on_subdomain;
+			const result = await fetchResult<{
+				available_on_subdomain: boolean;
+				id: string;
+				etag: string;
+				pipeline_hash: string;
+			}>(
+				workerUrl,
+				{
+					method: "PUT",
+					body: createWorkerUploadForm(worker),
+				},
+				new URLSearchParams({
+					include_subdomain_availability: "true",
+					// pass excludeScript so the whole body of the
+					// script doesn't get included in the response
+					excludeScript: "true",
+				})
+			);
+
+			available_on_subdomain = result.available_on_subdomain;
+
+			logger.debug("ID: ", result.id);
+			logger.debug("ETag: ", result.etag);
+			logger.debug("PipelineHash: ", result.pipeline_hash);
```

Comment:
will these fields be on every response?

Reply:
Ah your description says it won't be there. In which case can it log something other than a blank string? Something distinct, like "-Not available-". 

Please also add comments here warning not to change this since it'll possible be parsed.